### ISSUE TITLE
Promise.any false positive

### ISSUE THREAD

**Issue by Bauke** [dustinspecker/eslint-plugin-no-use-extend-native#134]
```js
async function example() {
  await Promise.any([]);
}

example();
```

```
  example.js:2:8
  ✖  2:8  Avoid using extended native objects  no-use-extend-native/no-use-extend-native

  1 error
```

**Comment by dustinspecker** [comment]
A new version of `obj-props` was just published (1.4.0). This should be fixed by re-installing `eslint-plugin-no-use-extend-native`. Thank you for the PR!
